configure

```diff
@@ -2121,6 +2125,11 @@ else
   else
   inf "        safe strings ............. no"
   fi
+  if test "$afl_instrument" = "true"; then
+  inf "        afl-fuzz always enabled .. yes"
```

Comment:
Do we really need the configure setting now that `$(opam config var lib)/ocaml/ocaml_compiler_internal_params` exists?